build.gradle

```diff
@@ -34,7 +34,7 @@ dependencies {
 
     // Ethereum
     implementation('org.web3j:web3j-spring-boot-starter:1.6.0')
-    implementation('org.web3j:core:5.0.0')
+    implementation('org.web3j:core:4.5.11')
```

Comment:
there is a runtime error with version over 4.5.11

Reply:
What kind of error? We have to always understand the reason and solve it. Downgrade the library version is not a solution.

Reply:
I keep getting this error when i try to getBlockByBlockNumber [http://i.imgur.com/LTow941.png](url)